alvr/client_openxr/src/interaction.rs

```diff
@@ -61,14 +63,14 @@ pub fn initialize_interaction(
     #[cfg(target_os = "android")]
     if let Some(config) = &face_tracking_sources {
         if (config.combined_eye_gaze || config.eye_tracking_fb)
-            && matches!(platform, Platform::Quest3 | Platform::QuestPro)
+            && matches!(platform, Platform::QuestPro)
```

Comment:
Missing the Quest 3 here seems wrong

Reply:
This is for eye gaze. the Quest 3 does not have eye tracking

Reply:
Huh, why was it there before then

Reply:
By mistake i guess. as for pico i left both Neo 3 and 4 because both have some eye tracking variants